taskcluster/ac_taskgraph/transforms/beetmover.py

```diff
@@ -31,7 +32,24 @@ def resolve_keys(config, tasks):
 
 @transforms.add
 def set_artifact_map(config, tasks):
+
+    def _craft_path_version(version, build_type, nightly_version):
+        """Helper function to craft the correct version to bake in the artifacts full
+        path section"""
+        ret = "{}{}".format(
```

Comment:
Nit: What does `ret` stand for?

Reply:
I renamed it to `path_version` so make things clearer.